src/operations.cpp

```diff
@@ -99,7 +99,8 @@
 
 #   include <sys/types.h>
 #   include <sys/stat.h>
-#   if !defined(__APPLE__) && !defined(__OpenBSD__) && !defined(__ANDROID__) && !defined(__VXWORKS__)
+#   if defined(__wasm)
```

Comment:
This would be nicer than the existing code base, but I'd honestly think that future readers would enjoy a comment here, especially if WASI should ever be extended to cover things like `stat`.

Reply:
You're right, of course--I lost the comment while porting the diff to `develop` from 1.72.0. Fixed now.

Reply:
Note that this is specifically `statfs` and `statvfs`, not `stat` (it does have `stat`). `statfs` gets filesystem statistics, not file statistics.